packages/backend/migrations/20211022210203_create_invoices_table.js

```diff
@@ -2,17 +2,32 @@ exports.up = function (knex) {
   return knex.schema.createTable('invoices', function (table) {
     table.uuid('id').notNullable().primary()
     table.uuid('accountId').notNullable()
-    table.foreign('accountId').references('accounts.id')
+    table.foreign('accountId').references('accounts.id').onDelete('CASCADE')
     table.uuid('paymentPointerId').notNullable()
     table.foreign('paymentPointerId').references('paymentPointers.id')
     table.boolean('active').notNullable()
     table.string('description').nullable()
     table.timestamp('expiresAt').nullable()
+    table.bigInteger('amountToReceive').nullable()
```

Comment:
I'm guessing these are `nullable` for spsp/web monetization?
What if we didn't use invoices for that?
We could put a web monetization account id in the paymentpointers table, and then make invoice `amountToReceive` and `expiresAt` notNullable

Reply:
I think the idea of spsp/WM being pseudo-invoices internally is that that is how we want them to appear from the user's perspective. i.e. if we are displaying a transaction history, it would show all of their invoices and WM payment received. And it's much easier to have a unified interface like that if they are all in one table.

Reply:
Maybe once we have invoice withdrawals set up, the wallet could just track and display those withdrawals instead of looking up invoices in Rafiki for transaction history.

Reply:
Also `amountToReceive` would be null for fixedSend as you wouldn't necessarily know as the sender how much the invoice should receive.